packages/aws-cdk/bin/cdk.ts

```diff
@@ -115,6 +115,39 @@ async function parseCommandLineArguments() {
           'which skips CloudFormation and updates the resources directly, ' +
           'and falls back to a full deployment if that is not possible. ' +
           'Do not use this in production environments',
+      })
+      .option('watch', {
+        type: 'boolean',
+        desc: 'Continuously observe the project files, ' +
+          'and deploy the given Stack(s) automatically when changes are detected. ' +
+          'Implies --hotswap by default',
+      }),
+    )
+    .command('watch [STACKS..]', "Shortcut for 'deploy --watch'", yargs => yargs
+      // I'm fairly certain none of these options, present for 'deploy', make sense for 'watch':
+      // .option('all', { type: 'boolean', default: false, desc: 'Deploy all available stacks' })
```

Comment:
`--all` is the default no?

Reply:
Yes, but I don't think a separate `--all` option makes much sense for `watch` - that's why I removed it.

Reply:
Related to the discussion about `watch` without any arguments. I would expect this to deploy all stacks.

Reply:
Yep, I get that. I'm just saying that `watch` deploying all Stacks is such a rarity, that I don't think it deserves a separate CLI option for that.

Reply:
OK.